### ISSUE TITLE
Flatpak update check error

### ISSUE THREAD

**Issue by marvasDE** [dhruv8sh/arch-update-checker#8]
When I install the **arch-update-checker**, it checks for updates forever. Also, the mini program shows the following error message:

```
Error code: 0
Error:Fehler: Ungültige Kennung : Name darf nicht leer sein
sed: -e Ausdruck #1, Zeichen 0: Kein vorheriger regulärer Ausdruck
```
(german error message)

If I set the default value of **flatpakEnabled** to **false** via main.xml, the **arch-update-checker** starts without issues.
If I enable flatpak in the settings, I get the same error and endless loading again.

Flatpak is installed and currently there are 0 updates available.
I use the current version 0.7.5 of the **arch-update-checker** from Github.

**Comment by dhruv8sh** [comment]
#5 and #4 had the same issues please confirm it's not the same problem.
This was fixed in the latest release anyways. Please update and let me know if it happens.

**Comment by marvasDE** [comment]
As I said, I already use the latest version (this main branch, tag **v0.7.5**)
It realy looks like #5, but I would like to use Flatpak and I have Flatpak packages installed (but none of them have updates right now).

Disabling a feature does not seem to be a solution for a feature, isn't it? :)

**Comment by dhruv8sh** [comment]
Unfortunately, this behavior is difficult to test for me since I would have to clone your setup.
Can you run this command and paste the output here?
```
upd=$(flatpak remote-ls --columns=name,application,version --app --updates | \
    sed 's/ /-/g' | sed 's/\t/ /g')
    output=""
    if [ -n "$upd" ]; then
        while IFS= read -r app; do
            id=$(echo "$app" | awk '{print $2}')
            ver=$(flatpak info "$id" | grep "Version:" | awk '{print $2}')
            output+="$(echo "$app $ver\n")"
        done <<< "$upd"
    fi
    echo -en "$output"
```
Also for
```
flatpak remote-ls --columns=name,application,version --app --updates
```

**Comment by marvasDE** [comment]
Both commands outputs nothing.

> Edit: I am still skeptical of the update you have made. Did you restart the plasmashell after updating?

Thank you!
It was right to be skeptical. No, I didn't restart and this seems to fix the issue.

It's weird, because I made changes like changing the main.xml and they were applied.
I tried `plasmapkg2 -u .` and also `plasmapkg2 -r .` + `plasmapkg2 -i .` and nothing fixed the issue, but a restart? Why? 🥲

Originally I installed it via Plasma itself. Was this perhaps the problem?
